packages/e2e-tests/specs/widgets/editing-widgets.test.js

```diff
@@ -888,14 +895,18 @@ async function saveWidgets() {
 	} );
 	await updateButton.click();
 
-	await findAll( {
+	// Expect the "Widgets saved." snackbar to appear.
+	const savedSnackbarQuery = {
+		role: 'button',
+		name: 'Dismiss this notice',
 		text: 'Widgets saved.',
-	} );
+	};
+	await expect( savedSnackbarQuery ).toBeFound();
 
-	// FIXME: The snackbar above is enough for the widget areas to get saved,
-	// but not enough for the widgets to get saved.
-	// eslint-disable-next-line no-restricted-syntax
-	await page.waitForTimeout( 500 );
+	// Close the snackbar.
+	const savedSnackbar = await find( savedSnackbarQuery );
+	await savedSnackbar.click();
+	await expect( savedSnackbarQuery ).not.toBeFound();
```

Comment:
Does it need to be dismissed for the test to work? If so, I think that's ok, but otherwise it should probably be in a separate test rather than something that happens any time widgets are saved.

Reply:
They should be dismissed, or else there will multiple snackbars on the page, and we won't know when the widgets are saved. Which was the reason it was flaky before 😅

Reply:
Ah, I see. If only the snackbar disappearing speed was proportional to the speed the e2e tests move. 😄 

Could consider adding a comment like 'avoids multiple snackbars present on the screen'.